js/read_dir_test.ts

```diff
@@ -22,12 +22,24 @@ function assertSameContent(files: FileInfo[]) {
   assertEqual(counter, 2);
 }
 
-test(function readDirSyncSuccess() {
+testPerm({ write: true, read: true }, function readDirSyncSuccess() {
```

Comment:
I think this test code does not use write permission. `write: true`

Reply:
good catch